dxm/src/main/java/io/pcp/parfait/dxm/PcpOffset.java

```diff
@@ -5,4 +5,5 @@
 
 interface PcpOffset {
```

Comment:
yeah I think the other interface might be a better way to go.  I agree with not using `length`, as soon as I submitted that comment I realised it was weak...

`byteLength` ?